adafruit_midi/midi_message.py

```diff
@@ -111,7 +111,7 @@ def channel(self):
     @channel.setter
     def channel(self, channel):
         if channel is not None and not 0 <= channel <= 15:
-            raise "Channel must be 0-15 or None"
+            raise "Channel must be 0-15 or None"  # pylint: disable=raising-bad-type
```

Comment:
This should be a `ValueError`, I think. 
```suggestion
            raise ValueError("Channel must be 0-15 or None")
```

Reply:
Ahh ok. Thanks! That makes a lot more sense